packages/button/src/css/index.ts

```diff
@@ -46,23 +38,28 @@ export default {
     alignItems: 'stretch',
     padding: `${layout.spacingSmall} ${layout.spacingMedium}`,
     border: 0,
-    borderRadius: '2px',
-    fontSize: type.fontSize200,
+    borderRadius: '3px',
+    fontSize: type.fontSize300,
     fontWeight: type.fontWeight500,
     textAlign: 'center',
     whiteSpace: 'nowrap',
     textDecoration: 'none',
-    transition: `all ${motion.speedNormal}`,
+    transition: `all ${motion.speedXFast}`,
```

Comment:
tangential note: been reading lately about the perf benefits of transitioning only the properties we are actually wanting changed. Then tweening of other properties is not attempted.

Reply:
changed transition from all to specific properties